project/Dependencies.scala

```diff
@@ -57,4 +57,14 @@ object Dependencies {
     "com.github.julien-truffaut" %% "monocle-core" % monocleVersion,
     "com.github.julien-truffaut" %% "monocle-macro" % monocleVersion
   )
+
```

Comment:
These aren't used in the project. Are they temporarily commented out in the `build.sbt` file or can we dump them altogether?